.gitignore

```diff
@@ -40,3 +40,6 @@ config/customization/*
 config/transifex.netrc
 !config/provider/.gitkeep
 !config/customization/.gitkeep
+
+# ignore IDE configuration 
+.idea/
```

Comment:
What IDE is this for?

Maybe this would better fit into a [global gitignore](https://help.github.com/articles/ignoring-files/#create-a-global-gitignore) on your machine?
If not it might be a minimal pull request on it's own as it's independent of the rest.